components/tool-navbar.vue

```diff
@@ -117,7 +156,6 @@
 
     .brand-styling {
```

Comment:
I think it's better to move all the styles to the external style sheet.

Reply:
So, I should have been clearer in my comment on your PR. Usually the separate stylesheet is for styles that will be used in multiple places in the app. Here, the styles are just for this component.

Reply:
I see your point. I guess this also comes down to personal preference so I'll leave the decision up to you.